helmfile.d/values/grafana/grafana-user.yaml.gotmpl

```diff
@@ -112,9 +112,15 @@ grafana.ini:
     client_secret: $__env{clientSecret}
     use_pkce: true
     scopes: {{ .Values.grafana.user.oidc.scopes }}
+    {{- if .Values.grafana.user.oidc.useInternalDex }}
```

Comment:
Might be simpler if this was conditional on `externalDexDomain` being set instead of having two settings?